<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Infomation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="minimumSize">
         <size>
          <width>420</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>config</string>
        </property>
        <widget class="QPushButton" name="pushButton_con_net">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>52</y>
           <width>96</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>ping network</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_discon_net">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>127</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>exit network</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonCamera">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>40</y>
           <width>112</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(204, 0, 0);</string>
         </property>
         <property name="text">
          <string>Camera</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonCar">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>70</y>
           <width>112</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="text">
          <string>Car</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_send">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>127</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>send UDP</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>ctrl</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonStop">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(164, 0, 0);</string>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="shortcut">
            <string>Space</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_down">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="text">
              <string>DOWN</string>
             </property>
             <property name="shortcut">
              <string>Down</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_left">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="text">
              <string>LEFT</string>
             </property>
             <property name="shortcut">
              <string>Left</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_up">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="text">
              <string>UP</string>
             </property>
             <property name="shortcut">
              <string>Up</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_right">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="text">
              <string>RIGHT</string>
             </property>
             <property name="shortcut">
              <string>Right</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
